lib/routes/sohumobile/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/limit',
+    categories: ['traditional-media'],
```

Comment:
`Traditional media` means newspapers, magazines, radio, and television broadcast and I don't think this site has any of these.